plugins/crate-copy-azure/src/test/java/io/crate/copy/azure/AzureCopyIntegrationTest.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.copy.azure;
+
+import static io.crate.testing.Asserts.assertThat;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Locale;
+
+import org.apache.lucene.tests.util.QuickPatchThreadsFilter;
+import org.elasticsearch.plugins.Plugin;
+import org.elasticsearch.test.IntegTestCase;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import com.carrotsearch.randomizedtesting.ThreadFilter;
+import com.carrotsearch.randomizedtesting.annotations.ThreadLeakFilters;
+import com.sun.net.httpserver.HttpServer;
+
+import io.crate.azure.testing.AzureHttpHandler;
+import io.crate.lucene.CrateLuceneTestCase;
+
+@ThreadLeakFilters(
+        defaultFilters = true,
+        filters = {
+            QuickPatchThreadsFilter.class,
+            AzureCopyIntegrationTest.OpenDALFilter.class,
+            CrateLuceneTestCase.CommonPoolFilter.class})
+public class AzureCopyIntegrationTest extends IntegTestCase {
+
+    private static final String CONTAINER_NAME = "test";
+    private static final String AZURITE_ACCOUNT = "devstoreaccount1";
+    private static final String AZURITE_KEY =
+            "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==";
```

Comment:
To clarify that this is just a dummy.
```suggestion
            "dummy-key;
```